designguidelines.md

```diff
@@ -0,0 +1,36 @@
+#### Themes
+
+* __default__: teal outline
+* __standout__: solid teal
+* __uncolored__: monochrome
+* __inverse__: for use on dark backgrounds
+* __b2c__: A theme for b2c products eg http://help.ft.com
+
+and the following sizes ([documentation](#size-modifiers)):
```

Comment:
Would this be better as `and the following sizes:` with the link removed, as it won't go back to the repo docs